edge/pkg/devicetwin/dtmodule/dtmodule_test.go

```diff
@@ -75,3 +80,135 @@ var _ = Describe("Dtmodule", func() {
 		})
 	})
 })
+
+func TestDTModule_InitWorker(t *testing.T) {
+	type fields struct {
+		Name string
+	}
+	ctx, err := dtcontext.InitDTContext()
+	assert.Nil(t, err)
+	recvCh, confirmCh, heartBearCh := make(chan interface{}), make(chan interface{}), make(chan interface{})
+	tests := []struct {
+		name   string
+		fields fields
+		want   DTWorker
+	}{
+		{
+			name: dtcommon.MemModule,
+			fields: fields{
+				Name: dtcommon.MemModule,
+			},
+			want: MemWorker{
+				Worker: Worker{
+					ReceiverChan:  recvCh,
+					ConfirmChan:   confirmCh,
+					HeartBeatChan: heartBearCh,
+					DTContexts:    ctx,
+				},
+				Group: dtcommon.MemModule,
+			},
+		},
+		{
+			name: dtcommon.TwinModule,
+			fields: fields{
+				Name: dtcommon.TwinModule,
+			},
+			want: TwinWorker{
+				Worker: Worker{
+					ReceiverChan:  recvCh,
+					ConfirmChan:   confirmCh,
+					HeartBeatChan: heartBearCh,
+					DTContexts:    ctx,
+				},
+				Group: dtcommon.TwinModule,
+			},
+		},
+		{
+			name: dtcommon.DeviceModule,
+			fields: fields{
+				Name: dtcommon.DeviceModule,
+			},
+			want: DeviceWorker{
+				Worker: Worker{
+					ReceiverChan:  recvCh,
+					ConfirmChan:   confirmCh,
+					HeartBeatChan: heartBearCh,
+					DTContexts:    ctx,
+				},
+				Group: dtcommon.DeviceModule,
+			},
+		},
+		{
+			name: dtcommon.CommModule,
+			fields: fields{
+				Name: dtcommon.CommModule,
+			},
+			want: CommWorker{
+				Worker: Worker{
+					ReceiverChan:  recvCh,
+					ConfirmChan:   confirmCh,
+					HeartBeatChan: heartBearCh,
+					DTContexts:    ctx,
+				},
+				Group: dtcommon.CommModule,
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			dm := &DTModule{
+				Name: tt.fields.Name,
+			}
+			dm.InitWorker(recvCh, confirmCh, heartBearCh, ctx)
+			if !reflect.DeepEqual(dm.Worker, tt.want) {
+				t.Errorf("InitWorker() = %v, want %v", dm.Worker, tt.want)
+			}
+		})
+	}
+}
+
+func TestDTModule_Start(t *testing.T) {
+	type fields struct {
+		Name        string
+		RecvCh      chan interface{}
+		ConfirmCh   chan interface{}
+		HeartBeatCh chan interface{}
+		Ctx         *dtcontext.DTContext
+	}
+	ctx, err := dtcontext.InitDTContext()
+	assert.Nil(t, err)
```

Comment:
Actually using `require.Nil` is better, but the error returned by the `InitDTContext` is always nil, so `require` may not be necessary
```go
"github.com/stretchr/testify/require"
```